### PR TITLE
Update the pest control process to align with the bugsnag redesign

### PR DESCRIPTION
**Pull request by jackdclark**
# What does this change?

Our pest control process has a _how to use bugsnag_ section that is now out of date because Bugsnag has redesigned.
# Why?

This section is useful for new starters

### CODE REVIEW

**Comment by hx-markterry** [comment]
ok with me :rocket:

**Comment by jamesmcneil** [comment]
Looks good to me, great job :+1:

File: pest-control-process.md [hunk 1]
```diff
@@ -55,43 +55,37 @@ A member of each pod is encouraged to attend to contribute to improving this pro
...
-You can use the search form on the left hand side to limit what bugs you see:
+You can use the filters on the left hand side to quickly limit what bugs you see, the **introduced today** filter is particularly helpful for the pest control process.
 
 ![](/images/pest-control-process/bug-filter.png)
 
-##### Ungrouped View
-Ungrouped view shows you a live error by error view of the project. It also shows you more information regarding each bug such as user and device settings.
+You can also use the search bar for more complex filtering, allowing you to search by **error message**, **release stage**, **browser version** and various other things.
+
+_Tip: searching by error message can be useful for working out the number of users impacted as Bugsnag's grouping isn't the most reliable and often leaves the same errors ungrouped._
```
Comment (pmcnr-hx) [inline pest-control-process.md]: I _abuse_ emoticons, but a :information_source: here would look nice. :wink:

**Comment by pmcnr-hx** [comment]
Good job! :+1:

**Comment by jackdclark** [comment]
:information_source: added for you @pmcnr-hx :smile:

**Comment by pmcnr-hx** [comment]
Much emoji! :shipit:

**Pull request merged**